pyaedt/desktop.py

```diff
@@ -533,99 +658,212 @@ def __init__(
             if "oDesktop" in dir(self._main):
                 del self._main.oDesktop
-            self._main.student_version, version_key, version = self._set_version(specified_version, student_version)
-            settings.aedt_version = version_key
-            settings.is_student = self._main.student_version
-            if not new_desktop_session and not is_ironpython:  # pragma: no cover
-                sessions = active_sessions(
-                    version=version_key, student_version=student_version, non_graphical=non_graphical
-                )
-                if aedt_process_id:
-                    for session in sessions:
-                        if session[0] == aedt_process_id and session[1] != -1:
-                            self.port = session[1]
-                            settings.use_grpc_api = True
-                            break
-                        elif session[0] == aedt_process_id:
-                            settings.use_grpc_api = False
-                            break
-
-            if version_key < "2022.2":
-                settings.use_grpc_api = False
-            elif (
-                version_key.startswith("2022.2")
-                and not self.port
-                and not self.machine
-                and settings.use_grpc_api is None
-                and _com != "gprc_v3"
-            ):
-                settings.use_grpc_api = False
-            elif settings.use_grpc_api is None or _com == "gprc_v3":
-                settings.use_grpc_api = True
-            if _com == "ironpython":  # pragma: no cover
+            if starting_mode == "ironpython":
                 self._logger.info("Launching PyAEDT outside AEDT with IronPython.")
                 self._init_ironpython(non_graphical, new_desktop_session, version)
-            elif settings.use_grpc_api:
-                settings.use_grpc_api = True
-                self._init_cpython_new(
-                    non_graphical, new_desktop_session, version, self._main.student_version, version_key
-                )
-            elif _com == "pythonnet_v3":
+            elif starting_mode == "com":
                 self._logger.info("Launching PyAEDT outside AEDT with CPython and PythonNET.")
-                self._init_cpython(
+                self._init_dotnet(
                     non_graphical,
                     new_desktop_session,
                     version,
-                    self._main.student_version,
+                    student_version_flag,
                     version_key,
                     aedt_process_id,
                 )
-            else:
-                from pyaedt.generic.clr_module import win32_client
+            elif starting_mode == "grpc":
+                self._logger.info("Launching PyAEDT outside AEDT with gRPC plugin.")
+                self._init_grpc(non_graphical, new_desktop_session, version, student_version_flag, version_key)
+
+        # # console di Ironpython dentro AEDT
```

Comment:
```suggestion
        # # console di IronPython dentro AEDT
```
Use English rather than Italian.